### DISCUSSION TITLE
NORAM MessageBird Compliance 🇺🇸🇨🇦 [for Attestation Service Operators]

### DISCUSSION THREAD

**Discussion by arthurgousset** [celo-org/social-connect#241] [News (Updates)]
## NORAM MessageBird Compliance

Hi attestation service operators,

As [`daluxx | StakesStone`](https://discord.com/channels/600834479145353243/654086275900375041/996661712256245860) kindly highlighted on Discord, MessageBird recently sent out an email requesting attestation service operators submit a form to comply with new developments in the American and Canadian markets:

> all toll-free messaging services must complete a mandatory compliance verification process. Numbers that are in use and unverified will need to complete verification prior to September 30, 2022. 
> 
> In order for MessageBird to meet this deadline on behalf of our customers, we’re asking that you please submit your details to us as soon as possible.

## What next?

We have pre-populated a short form for your convenience and left information that is specific to your attestation service blank.

Please:

1. [Download the short form we pre-populated](https://docs.google.com/document/d/1KF3O1zoKKL5SYlf-wgzk2lfBMdidoBK3gXyrBlYQxuU/edit?usp=sharing) and fill in the missing details marked with brackets `[…]` (e.g. contact information and toll-free numbers)
2. Submit a ticket to MessageBird [Support Team](https://support.messagebird.com/hc/en-us/requests/new?ticket_form_id=1900000387213&utm_campaign=PM-170+-+TFN+Verification+Notice+2022&utm_medium=email&utm_source=autopilot) with the completed form
3. [Submit your `validator name` in this Google Form](https://forms.gle/SHRztUePVRgry7Pd8) so I can track who completed the MessageBird compliance form

Thank you and feel free to ping us with any questions!
The cLabs identity team

**Comment by Creamers158** [comment]
**Finally got a reply from messagebird after few weeks of waiting:**

Your patience has been appreciated. 

Unfortunately, the use-case could not be verified because it conflicts with the Disallowed Content Policy: **Cryptocurrency**
 
Thank you.
Best from Dar es Salaam,

Eric Kaika | Technical Support Specialist.

**Reply by Creamers158** [reply]
**Messagebird response when I said it was about a verification code:**

Thanks for reaching out.
 
To advise, the use case was not verified because of the 'content category' and not just the message content itself.
 
The Provider had expanded the list of categories for their 'Disallowed Content Policy' to align with wireless carrier policies and agreements. The following content 'CryptoCurrency' has been identified as part of this list and will not be supported on the provider's network, thus the non-verification.
 
Hope this information helps.

**Reply by arthurgousset** [reply]
Thanks for following up here @Creamers158 🙏 As [mentioned on Discord](https://discord.com/channels/600834479145353243/654086275900375041/1011944164264529940), it's interesting to see that the same form led to different outcomes. Let me contact MessageBird to get some more context and clarify the responses. 

I'll get back to you ✅

**Comment by y3v63n** [comment]
**Got a similar reply from them:**

Hi,

Thank you very much for your patience.

I got an update from the supplier, that unfortunately, they cannot verify this traffic on our network because it conflicts with our Disallowed Content Policy:
High-Risk Financial Services - Cryptocurrency

Please refer to the Disallowed Content Policy below for reference.

**Comment by arthurgousset** [comment]
Copying [`mbay2002 |  VCAP` response](https://discord.com/channels/600834479145353243/654086275900375041/1012756564517593149) from Discord (for future reference):

> FWIW: I was approved by messagebird toll free long ago, and for an explanation I wrote "user enters their phone number to request passcode for 2FA verification". That was approved.
> 
> My understanding is now they blanket reject (by internal policy) any use case that mentions crypto or blockchain. So I would suggest using the language I have above.

**Comment by arthurgousset** [comment]
**UPDATE**

Thanks to everyone's feedback, I made some changes to the form and would recommend you resubmit the modified form if your previous submission was rejected.
I removed the 2 mentions of "blockchain" and replaced some descriptions with the wording `mbay2002 |  VCAP` successfully used. 

For completeness the modified sections now say:

**Summary of the use-case**: 

> User enters their phone number to request passcode for 2FA verification

**How will consumers be opting-in to SMS programs?**:

> Consumers opt-in by registering using their phone number and receive an SMS passcode for 2FA verification.

👇 **In summary, please** 👇 

 1. [Download the short modified form we pre-populated](https://docs.google.com/document/d/1KF3O1zoKKL5SYlf-wgzk2lfBMdidoBK3gXyrBlYQxuU/edit?usp=sharing) and fill in the missing details marked with brackets `[…]` (e.g. contact information and toll-free numbers)
2. Submit a ticket to MessageBird [Support Team](https://support.messagebird.com/hc/en-us/requests/new?ticket_form_id=1900000387213&utm_campaign=PM-170+-+TFN+Verification+Notice+2022&utm_medium=email&utm_source=autopilot) with the completed form

☝️ ☝️ ☝️ ☝️ ☝️ ☝️ ☝️ ☝️

**Reply by arthurgousset** [reply]
_Also [shared on Discord](https://discord.com/channels/600834479145353243/654086275900375041/1015215549208350780) for better visibility_

**Reply by Creamers158** [reply]
I received my confirmation after resubmitting the adjusted form 7 weeks ago:
_Please be informed that the usecase for TFN: 183xxxxxxxx has been verified and is ready for use._
